addons/planner_project/static/src/less/planner_project.less

```diff
@@ -0,0 +1,62 @@
+
+#PlannerModal {
```

Comment:
I guess this style is specific for planner project. To avoid conflict with other planner (since the page has the same name), add a class `planner_xxxxx` (where xxxx is the planner name, such as in the selection field of the planner model) on the tag `#PlannerModal`.